feature_importance.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
+import pickle
+import pandas as pd
+from sklearn.ensemble import RandomForestClassifier
+import matplotlib.pyplot as plt
+from matplotlib.ticker import MultipleLocator, PercentFormatter
+
+
+def plot_importances(models, model_names, labels=None, y_limit=(0,0.5)):
+
+    """Plot feature importances of random forest models.
+
+    Args:
+        models: Model object or a list of model objects.
+        model_names: Model name or a list of name for each model.
+        labels: A list of feature name that x labels show in this order. 
+        y_limit: The limit of y axis. 
+
+    Return:
+        fig: A matplotlib.figure.Figure object.
+    """
+
+    if isinstance(models, RandomForestClassifier):
```

Comment:
Is this fn defined somewhere?

Reply:
It is a built-in python function. I use it to check `models` if it is a classifier object and make it a list. So the user can simply pass a model object to `plot_importances`.